packages/paste-website/src/pages/components/combobox/index.mdx

```diff
@@ -401,102 +341,57 @@ yarn add @twilio-paste/combobox - or - yarn add @twilio-paste/core
+| `inputValue?`               | `string`                                              | Sets the value of the input inside the Combobox.                                                                                                          | `''`           |
+| `selectedItem?`             | any                                                   | Used to set the selected item of the combobox.                                                                                                            |                |
+| `initialIsOpen?`            | `boolean`                                             | Whether the Combobox is open on initial render.                                                                                                           | `false`        |
+| `initialSelectedItem?`      | `string \| Record<string, unknown>`                   | Sets the initial item selected when a Combobox is initialized.                                                                                            | `[]`           |
+| `disabledItems?`            | `any[]`                                               | An array of items that should be disabled.                                                                                                                | `[]`           |
+| `emptyState?`               | `React.FC`                                            | A custom empty state component to render when there are no items in the list.                                                                             | `null`         |
+| `onSelectedItemChange?`     | `(newSelectedItem: any[]) => void`                    | Callback function for after an item is selected or deselected.                                                                                            |                |
+| `onHighlightedIndexChange?` | `(newHighlightedIndex: number) => void`               | Callback function for after an item is highlighted. Items are highlighted if they are hovered over or with keyboard actions.                              |                |
+| `onInputValueChange?`       | `(newInputValue: string) => void`                     | Callback function for after the input value is changed.                                                                                                   |                |
+| `onIsOpenChange?`           | `(newIsOpen: boolean) => void`                        | Callback function for after the Combobox is opened or closed.                                                                                             |                |
+| `state?`                    | `UseComboboxPrimitiveReturnValue<any>`                | Used with the `useCombobox` hook. When using this prop, all other state props are **ignored**. They must be passed to `useCombobox` as arguments instead. |                |
 
 #### useCombobox arguments
 
```

Comment:
Can you add a Figma section (as an H3) right before the Changelog, with this content?

https://docs.google.com/document/d/143idKgnfqFCmSceZZ67nIcLCxRnBsuxYPB8TMxTxKno/edit#bookmark=kix.ih4ca8a5a8co